content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,28 @@ menu:
     parent: About the project
 ---
 
+## v1.8.4 [unreleased]
+
+The InfluxDB Enterprise 1.8.4 release builds on the InfluxDB OSS 1.8.4 release.
```

Comment:
There is some question whether we will have a 1.8.4 OSS release.  Tim said it was a "soft" release.  Check with Tim and Ryan about whether we are putting the 1.8.4 OSS release out.

Reply:
Thank you, @davidby-influx - 

@timhallinflux @rbetts @psteinbachs, with OSS 1.8.4 being a "soft" release, are we posting binaries on download portal and shall we include in the docs?

Reply:
Yes we will have a 1.8.4 OSS release.  "Soft" in this case is referring to whether or not this will be the last 1.8.x OSS release, and depends on continued 2.x OSS adoption numbers as well as how prepared we are to message/signal this position.